tensorflow/python/framework/tensor_shape_test.py

```diff
@@ -168,6 +168,18 @@ def testInequality(self):
     self.assertIs(None, tensor_shape.Dimension(None) != None)  # pylint: disable=g-equals-none
     self.assertTrue(tensor_shape.Dimension(12) != 12.99)
 
+  def testCompatibleWithInt(self):
+    self.assertEqual(5 + tensor_shape.Dimension(3),
+                     tensor_shape.Dimension(5) + 3)
```

Comment:
These tests just verify that `int + Dimension` and `Dimension + int` give the same result, but they don't check the actual values.

Can you split these each of these asserts into two, comparing against explicitly constructed Dimension objects?

Reply:
Thanks for your suggestion. Yes, I'd like to add it later.